src/sched/job_runner.cc

```diff
@@ -185,13 +185,7 @@ JobRunner::JobRunner(JobRunner::Config config) : config_(std::move(config)) {
 
 JobRunner::~JobRunner() {
     ready_for_stealing_.store(false);
```

Comment:
Should we set this for every call to Stop()?

Reply:
Not needed. `Stop` will not destroy the job queues so technically we can still steal jobs.